pkg/api/types.go

```diff
@@ -551,6 +553,20 @@ type RBDVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// MetadataVolumeSource represents a volume containing metadata about a pod.
+type MetadataVolumeSource struct {
```

Comment:
@pmorie I'm getting rid of all `metadata` reference in the code and docs replacing it with DownwardAPI and downward API.